tokio-util/src/task/spawn_pinned.rs

```diff
@@ -206,6 +361,16 @@ impl LocalPool {
             }
         }
     }
+
+    fn find_worker_by_idx(&self, idx: usize) -> (&LocalWorkerHandle, JobCountGuard) {
+        let worker = &self.workers[idx];
+        let task_count = worker.task_count.load(Ordering::SeqCst);
+        worker
+            .task_count
+            .fetch_add(task_count + 1, Ordering::SeqCst);
+
+        (worker, JobCountGuard(Arc::clone(&worker.task_count)))
+    }
```

Comment:
The `fetch_add` operation _adds_ to the value. Can you please add a test that would have caught this mistake?
```suggestion
    fn find_worker_by_idx(&self, idx: usize) -> (&LocalWorkerHandle, JobCountGuard) {
        let worker = &self.workers[idx];
        worker.task_count.fetch_add(1, Ordering::SeqCst);

        (worker, JobCountGuard(Arc::clone(&worker.task_count)))
    }
```

Reply:
Sorry, should have been more careful there. Updated an existing test that would have caught that.